src/Jackalope/Version/VersionHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Jackalope\Version;
+
+use Jackalope\Node;
+use Jackalope\Transport\TransportInterface;
+use PHPCR\UnsupportedRepositoryOperationException;
+
+class VersionHandler
+{
+    const MIX_VERSIONABLE = 'mix:versionable';
+    const MIX_SIMPLE_VERSIONABLE = 'mix:simpleVersionable';
```

Comment:
NodeTypeInterface has constants for these, but not using the namespace alias but the full namespace: https://github.com/phpcr/phpcr/blob/master/src/PHPCR/NodeType/NodeTypeInterface.php#L257